python/tests/sql/test_function.py

```diff
@@ -96,6 +96,9 @@ def test_st_buffer(self):
         function_df = self.spark.sql("select ST_Buffer(polygondf.countyshape, 1) from polygondf")
         function_df.show()
 
+        function_df = self.spark.sql("select ST_Buffer(polygondf.countyshape, 10, 'endcap=square') from polygondf")
+        function_df.show()
```

Comment:
Please do not do `show` in tests. If the old tests use `show`, remove them as well. Use assertion to check the correctnesst.

Reply:
Got it. I didn't know.